### PR TITLE
Error collection optin

### PR DESCRIPTION
**Pull request by faboweb**
> Thank you a lot for contributing to Cosmos Voyager!

### Issue

closes: #552

Missing:
- [x] IPC - I want to integrate electron.IPC first, as this would need another endpoint in the relay_server and I think it is ugly. #572 
- [x] tests

### Screenshots

image
image
image

❤️ Thank you!

### CODE REVIEW

File: app/src/renderer/components/common/NiSessionImport.vue [hunk 1]
```diff
@@ -38,21 +38,28 @@
         placeholder="Must be exactly 12 words")
       form-msg(name='Seed' type='required' v-if='!$v.fields.importSeed.required')
 
+    form-group(field-id="error-collection" field-label=' ')
+      .ni-field-checkbox
+        .ni-field-checkbox-input
+          input#sign-up-warning(type="checkbox" v-model="fields.errorCollection")
+        label.ni-field-checkbox-label(for="error-collection")
+          | I help development of Voyager by sending automatic error reports. This can be turned off at any time.
```
Comment (jolesbi) [inline app/src/renderer/components/common/NiSessionImport.vue]: "I'd like to opt in for remote error tracking to help improve Voyager"

**Comment by NodeGuy** [comment]
I'm not ready to assess these kinds of changes yet.

File: app/src/renderer/components/common/NiSessionImport.vue [hunk 2]
```diff
@@ -38,6 +38,13 @@
         placeholder="Must be exactly 12 words")
       form-msg(name='Seed' type='required' v-if='!$v.fields.importSeed.required')
 
+    form-group(field-id="error-collection" field-label=' ')
+      .ni-field-checkbox
+        .ni-field-checkbox-input
+          input#sign-up-warning(type="checkbox" v-model="fields.errorCollection")
+        label.ni-field-checkbox-label(for="error-collection")
+          | I'd like to opt in for remote error tracking to help improve Voyager
```
Comment (jolesbi) [inline app/src/renderer/components/common/NiSessionImport.vue]: i'm sorry this is annoying - but there should be a period at the end of this sentence.

File: app/src/renderer/components/common/NiSessionSignUp.vue [hunk 3]
```diff
@@ -52,6 +52,14 @@
         label.ni-field-checkbox-label(for="sign-up-backup")
           | I have securely written down my seed.
       form-msg(name='Backup confirmation' type='required' v-if='!$v.fields.signUpBackup.required')
+
+    form-group(field-id="error-collection" field-label=' ')
+      .ni-field-checkbox
+        .ni-field-checkbox-input
+          input#sign-up-warning(type="checkbox" v-model="fields.errorCollection")
+        label.ni-field-checkbox-label(for="error-collection")
+          | I'd like to opt in for remote error tracking to help improve Voyager
```
Comment (jolesbi) [inline app/src/renderer/components/common/NiSessionSignUp.vue]: and this one.

**Comment by faboweb** [comment]
Done @jolesbi. You are also always aloud to change those typing issues yourself.

**Pull request merged**